vscode/src/debugger/activate.ts

```diff
@@ -74,6 +74,14 @@ function registerCommands(context: vscode.ExtensionContext) {
     config: { name: string; [key: string]: any },
     options?: vscode.DebugSessionOptions,
   ) {
+    if (
+      vscode.debug.activeDebugSession !== undefined &&
+      vscode.debug.activeDebugSession.type === "qsharp"
+    ) {
+      // Multiple debug sessions disallowed, to reduce confusion
+      return;
+    }
```

Comment:
```suggestion
    if (vscode.debug.activeDebugSession?.type === "qsharp") {
      // Multiple debug sessions disallowed, to reduce confusion
      return;
    }
```

Reply:
I'm so happy to be taking fancy TypeScript syntax suggestions from you 😉